### PR TITLE
SIEVE paper summary

### PR DESCRIPTION
**Pull request by bryli**
Finished the summary of the SIEVE paper! Comments/edits are welcome :)

### CODE REVIEW

File: content/posts/sieve.md [hunk 1]
```diff
@@ -9,4 +9,74 @@ author: Bryan Li
...
+
+SIEVE uses a single FIFO queue and "hand" pointer.
+
+The queue maintains insertion order, and new objects are added to the head of the queue.
+
+Every object in the queue uses 1 bit to track visited/non-visited status, with the hand pointing to the next eviction candidate. The hand pointer moves from the tail to the head.
+
+- **Cache hits** set the visited bit of the cache line to 1.
+  - This also happens to be lock-free, which improves throughput.
+- **Cache misses** look at the head pointer and set the visited bit to 0 if it is 1, moving the hand pointer.
```
Comment (Ishaandham19) [inline content/posts/sieve.md]:
```suggestion
- **Cache misses** look at the hand pointer and set the visited bit to 0 if it is 1, moving the hand pointer.
```

File: content/posts/sieve.md [hunk 2]
```diff
@@ -9,4 +9,74 @@ author: Bryan Li
...
+  - Otherwise (`visited=0`) it will remove the eviction candidate from the cache, and move the hand pointer.
+
```
Comment (Ishaandham19) [inline content/posts/sieve.md:52]:
```suggestion
- The object is added to the head of the queue with bit set to 0.
```

I know this is mentioned above, but might be nice to specify that the bit is set to 0 initially?

File: content/posts/sieve.md [hunk 3]
```diff
@@ -9,4 +9,74 @@ author: Bryan Li
...
+This fulfills the idea of 'lazy promotion' by only promoting at eviction time, by keeping the object at the original location. Because "survived" objects are generally more popular than evicted ones, they will likely be accessed in the future as well. (similar to CLOCK algorithm).
+
+Then, the hand can quickly move towards the head where new objects are inserted, examining new objects shortly after admission. This achieves 'quick demotion'.
+
```
Comment (Ishaandham19) [inline content/posts/sieve.md:56]: Could add this visualization somewhere - https://cachemon.github.io/SIEVE-website/assets/images/illustrations/sieve_diagram.gif

**Review by Ishaandham19 — APPROVED** [review]
Thanks!
Left some minor suggestions

**Comment by bryli** [comment]
Thanks for reviewing! Will add these suggestions later today and then we can merge :)

**Pull request merged**